addon/objects/uml-primitives/fd-uml-package.js

```diff
@@ -0,0 +1,78 @@
+/**
+  @module ember-flexberry-designer
+*/
+
+import Ember from 'ember';
+
+import { BaseClass } from './fd-uml-class';
+import FdUmlElement from './fd-uml-element';
+
+/**
+  An object that describes a package on the UML diagram.
+
+  @class FdUmlPropertyObject
+  @extends FdUmlElement
+*/
+export default FdUmlElement.extend({
+
+  /**
+    The name of a package.
+
+    @property name
+    @type String
+  */
+  name: Ember.computed.alias('primitive.Name.Text'),
+
+  /**
+    Package's attributes.
+
+    @property attributes
+    @type String
+  */
+  attributes: Ember.computed.alias('primitive.Prop.Text'),
```

Comment:
Is this property so simple? In other objects it looks a little more complicated.

Reply:
Yes, it works this way